cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -146,6 +146,47 @@ def get_initial_simplex_and_names(
         """Return an initial Nelder-Mead simplex and the names for each parameter."""
 
 
+def _try_defaults_from_unitary(gate: 'cirq.Gate') -> Optional[Dict[str, 'cirq.TParamVal']]:
+    u = protocols.unitary(gate, default=None)
+    if u is None:
+        return None
+
+    gamma = np.angle(u[1, 1] * u[2, 2] - u[1, 2] * u[2, 1]) / -2
+    phi = -np.angle(u[3, 3]) - 2 * gamma
+    phased_c_theta_2 = u[1, 1] * u[2, 2]
+    if phased_c_theta_2 == 0:
+        # The zeta phase is multiplied with cos(theta),
+        # so if cos(theta) is zero then any value is possible.
+        zeta = 0
+    else:
+        zeta = np.angle(u[2, 2] / u[1, 1]) / 2
```

Comment:
not sure from the cirq side, but in pyle, we prefer not to return a value from [0, pi]  range since several important class of gates are around 0, a small disturb may cause a branch jump. So it is more prefer to have a value range from something like [-pi/4, 3*pi/4].

Reply:
np.angle returns angles in the $(-\pi, \pi]$ range https://numpy.org/doc/stable/reference/generated/numpy.angle.html#numpy-angle

Reply:
Oh, i see.